Code/Tools/FBuild/FBuildCore/Helpers/SLNGenerator.cpp

```diff
@@ -286,7 +286,7 @@ void SLNGenerator::WriteProjectConfigurationPlatforms( const Array< AString > &
     for( const AString * it = projectGuids.Begin() ; it != projectGuidsEnd ; ++it )
     {
         // only one project active in the solution build
-        const bool projectIsActive = solutionBuildProjectGuids.Find( *it );
+        const bool projectIsActive = solutionBuildProjectGuids.Find( *it ) != nullptr;
```

Comment:
This wasn't supposed to be in this PR, but I added it because it fixes a warning